### ISSUE TITLE
Whose immortan un-lives are over?

### ISSUE THREAD

**Issue by eyalroz** [wesnoth/wesnoth#4865]
Wesnoth 1.15.3
Rise of Wesnoth
Scenario 6: Back to Oldwood
The Wesfolk Lady says:

>  I am the Lady Jessene, a noble, a princess of the Wesfolk! I was betrayed by the Lich-Lords' dark pact with the orcs, in no small part because your fool Prince of Southbay convinced them that **their immortal un-lives were over!** 

Whose immortal un-lives? The Liches, or Orcs which are intended to have immortal un-lives?

If it's the Liches - how can they be convinced of that? They're standing and present, clearly their unlives aren't over. If it's the Orcs - would they really discuss the finer point of philosophy with a Human enemy?

Regardless, why would the Lich-Lords betray Jessene because either they or the Orcs got convinced that "their immortal un-lives are over"?

It seems to me this piece of dialogue needs some clarification.

**Comment by ln-zookeeper** [comment]
It's referring to the Prince of Southbay going to war against the Lich-Lords.

How I read it is pretty much like this: "by massing an army and attacking the Lich-Lords, your fool Prince of Southbay convinced them that their immortal un-lives will be over unless they do something about it, such as opening gates to letting hordes of orcs to the isle, which counts as betraying the Wesfolk because the orcs are just going to kill them too".

**Comment by eyalroz** [comment]
Wait wait, this doesn't make sense. At this point in the story, what we know is that an Orcish hoard is attacking, and the Prince of South Bay has amassed some forces to drive them away. We have no indication that the arrival of the Orcs is the result of a retaliatory measure by the Lich-Lords.

**Comment by ln-zookeeper** [comment]
No, the new Prince of Southbay attacks the Wesfolk and their Lich-Lords first, that's in the campaign's intro. In response, the Lich-Lords open the gates to let hordes of orcs through as a retaliatory measure, it's explained at the end of the first scenario. And that's what the line you quoted is referring to.

**Comment by eyalroz** [comment]
Oh, you mean _that_. So what gets me confused is the news Jenesse brings from the South about the huge Orc army seems to be of something that was _not_ the case at the beginning of the campaign.

I believe there's too much mental gymnastics to figure out the phrase I emboldended above. Can you possibly make it more explicit? e.g. something like: "... fool Prince of Southbay attacked us, convincing the Liches that  that their immortal un-lives were in danger!"

**Comment by ln-zookeeper** [comment]
> Oh, you mean _that_. So what gets me confused is the news Jenesse brings from the South about the huge Orc army seems to be of something that was _not_ the case at the beginning of the campaign.

Well, yeah, the orcs have advanced across and around the isle throughout the campaign at that point. So, yes, it's news that the main host of orcs is where she says it is.

> I believe there's too much mental gymnastics to figure out the phrase I emboldended above. Can you possibly make it more explicit? e.g. something like: "... fool Prince of Southbay attacked us, convincing the Liches that that their immortal un-lives were in danger!"

Well, honestly, since it's been that way for 15+ years and IIRC no one's complained about it being unclear so far, I'd be hesitant to change it just because of one person. But who knows, maybe there's plenty of others too.

**Comment by eyalroz** [comment]
@ln-zookeeper : Very few people take the time to scrutinize unclear dialog. Most of them just wait for the campaign to play out and things become clearer. And most of the rest wouldn't think of nitpicking with GitHub issues...

but - if you're still hesitant, how about a survey on the wesnoth.org forum?

**Comment by Wedge009** [comment]
I interpreted the dialogue to be pretty much what zookeeper stated at the beginning of this issue, but I concede it _could_ be phrased/explained better, especially for readers whose first language isn't English (I wonder how the respective translators handled this).

Also, is TRoW part of the 'grand campaign rewrite'? It's possible this could be reworded or removed altogether anyway.

**Comment by Wedge009** [comment]
It's been nearly a year and a half since this was reported. I still think the way it's currently phrased makes sense, but I have an unfair advantage as a native Anglophone. If there's no objection, I'll close this shortly.

**Comment by eyalroz** [comment]
So, you'll add the clarification?

**Comment by Pentarctagon** [comment]
The way I interpret the quote: liches are undead, orcs aren't undead, therefore "immortal unlives" can only possibly be referring to the liches.

**Comment by Wedge009** [comment]
With respect to the _specific_ question on who/what the 'immortal un-lives' refers to, that appears to be quite clear to everyone.
https://github.com/wesnoth/wesnoth/blob/f8a41624e3022947b6d7d9ac6b0311c4a7d1fee4/data/campaigns/The_Rise_Of_Wesnoth/scenarios/01_A_Summer_of_Storms.cfg#L229

However, upon discussion with @nemaara I can see there is some inconsistency with the reasoning for the lich-lords to open the gates to the orcs:
* I considered Southbay, as the largest kingdom on the Green Isle, to be substantial and therefore their prince raising an army to go to war against the Wesfolk would be a credible threat to the lich-lords.
* nemaara/Yumi raises the point that it would probably only be someone of appreciable magical skill who could convince a lich their un-life was over.
* Also, Jevyan clearly doesn't think highly of Haldric and his refugee army later in the campaign, so why would they feel threatened by Southbay? (Although it could be argued that Southbay's initial army was much larger than the group that eventually flees to the Great Continent.)
* On the other hand, the people of the Green Isle were previously able to topple 'one of the greatest leaders of [the Wesfolk]':
https://github.com/wesnoth/wesnoth/blob/f8a41624e3022947b6d7d9ac6b0311c4a7d1fee4/data/campaigns/The_Rise_Of_Wesnoth/scenarios/05_The_Oldwood.cfg#L351

> [Yumi] I find it hard to believe that the prince of southbay would be able to convince a lich lord of anything about their powers
[Yumi] so it refers to the liches, and yet who's gonna convince a lich that their life is over?
[Yumi] only someone else of appreciable magical skill, I'd imagine

> [Yumi] southbay being a military threat to the lichlords might be reasonable
[Yumi] I think that would be valid
[Yumi] but that's a very different thing than the prince of southbay being able to convince a lich that its unlife was over, which is a personal power kind of thing

So for that reason, I've renamed the issue and let it stand for the time being. As indicated in #4866, I suggest you discuss your concerns among fellow players in https://forums.wesnoth.org//viewtopic.php?t=1510.

**Comment by eyalroz** [comment]
Will start the discussion, but it's still not "convincing" the Liches their unlives are over, but rather that their unlives are in peril, or may be over etc.

**Comment by stevecotton** [comment]
> However, upon discussion with @nemaara I can see there is some inconsistency with the reasoning for the lich-lords to open the gates to the orcs:

A question for @nemaara - are the canon Lich Lords the strong creatures described in the Irdya Canon, or are they the much weaker creatures described by the current canon text of TRoW?

**Comment by nemaara** [comment]
Liches have always been a lot stronger than regular humans, even magi. I think based on unit descriptions, I'd say it'd take an arch mage level of skill to really threaten them. I personally also always read the Lich Lords as a bit stronger than regular liches, could be debatable though.
